### DISCUSSION TITLE
[feature] labels inside the slider-thumb

### DISCUSSION THREAD

**Discussion by Mopholo** [simeydotme/svelte-range-slider-pips#72] [Formatting / Styling] [answered]
Really nice implementation!   

I haven't played with this much yet but just looking at examples it didn't look like this feature was present. 

**Describe the feature**
Add the ability to use an embedded label inside the slider-thumb ( see attached image ) that remains after releasing the thumb. 

**Explain it's value / reasoning**
When you release the thumb the tool-tip disappears and 'position' becomes nonspecific.

While you could use labeled-pips to solve this there are uses where the labels might take up to much room without overlapping based on the length of the slider ( a few of your examples when viewed on mobile suffer from this ) 

**Additional context**
The date range slider is one example where this could work nicely, I think this is a very nice alternative or supplement to pips and tooltips.

NOTE: This might require allowing the user to changing the size of thumb's if that's not already possible 

![CustomPip](https://user-images.githubusercontent.com/53452067/170531903-26f0614a-eca2-489e-89a9-b394efb4afd4.jpg)

**Comment by simeydotme** [comment, marked as answer]
Hi @Mopholo , thanks for your suggestion!
It's totally possible with a little bit of CSS, and a formatter function;
https://svelte.dev/repl/7118f51fe91b4964b3cb89fc0fc229a8?version=3.46.4

I really should add this example to the Docs, haha.
